src/misc/misc.c

```diff
@@ -161,6 +161,24 @@ void print_dims(int D, const long dims[D])
 
 
 
+void debug_print_bits(int dblevel, int D, unsigned int bitmask)
+{
+	bool dbl = debug_logging;
+	debug_logging = false;
+	debug_printf(dblevel, "[");
+
+	 unsigned int maxPow = 1;
+
```

Comment:
white space looks wrong

Reply:
I don't know which white space you mean here. The indentation was indeed wrong for some reason.